core/templates/dev/head/pages/exploration_editor/ExplorationSaveService.js

```diff
@@ -224,7 +224,6 @@ oppia.factory('ExplorationSaveService', [
             '/pages/exploration_editor/' +
             'confirm_discard_changes_modal_directive.html'),
           backdrop: 'static',
-          keyboard: false,
```

Comment:
We don't want this modal to get closed with keyboard esc key! (as it's something which speaks about reloading exploration!)